.pipelines/diabetes_regression-ci.yml

```diff
@@ -186,7 +186,35 @@ stages:
           python -m ml_service.util.create_scoring_image --output_image_location_file image_location.txt
           # Output image location to Azure DevOps job
           IMAGE_LOCATION="$(cat image_location.txt)"
+          export IMAGE_TAGGED="$(sed 's/@sha256.*/:latest/g' image_location.txt)"
           echo "##vso[task.setvariable variable=IMAGE_LOCATION]$IMAGE_LOCATION"
+          echo "##vso[task.setvariable variable=IMAGE_TAGGED]$IMAGE_TAGGED"
+    - task: AzureResourceGroupDeployment@2
+      inputs:
+        azureSubscription: '$(AZURE_RM_SVC_CONNECTION)'
+        action: 'Create Or Update Resource Group'
+        resourceGroupName: '$(RESOURCE_GROUP)'
+        location: $(LOCATION)
+        templateLocation: 'Linked artifact'
+        csmFile: '$(Build.SourcesDirectory)/environment_setup/arm-templates/aci-password.json'
+        overrideParameters: '-baseName $(BASE_NAME)'
+        deploymentMode: 'Incremental'
+        deploymentOutputs: 'ACRCREDENTIALS'
+      displayName: 'Get ACR username and password'
```

Comment:
I haven't looked much into this, but I wonder if this can be achieved with the CLI? https://docs.microsoft.com/en-us/cli/azure/acr/credential?view=azure-cli-latest#az-acr-credential-show

Reply:
Yes, I think it will look nicer and remove the need for arm template.